SPECS/mm-common/mm-common.spec

```diff
@@ -1,6 +1,6 @@
 Summary:        mm-common module
 Name:           mm-common
-Version:        1.0.4
+Version:        1.0.5
```

Comment:
1.0.6 is also available at: https://download.gnome.org/sources/mm-common/1.0/mm-common-1.0.6.tar.xz

Reply:
Hi @xordux, Thanks for the comment. I had updated the package to the latest version.